Upstream: No, Alpine Linux specific
Reason: Uses proper path for oh-my-zsh

--- a/templates/minimal.zshrc
+++ b/templates/minimal.zshrc
@@ -1,4 +1,4 @@
-export ZSH="$HOME/.oh-my-zsh"
+export ZSH=/usr/share/oh-my-zsh
 ZSH_THEME="robbyrussell"
 plugins=(git)
 
--- a/templates/zshrc.zsh-template
+++ b/templates/zshrc.zsh-template
@@ -2,7 +2,7 @@
 # export PATH=$HOME/bin:$HOME/.local/bin:/usr/local/bin:$PATH
 
 # Path to your Oh My Zsh installation.
-export ZSH="$HOME/.oh-my-zsh"
+export ZSH=/usr/share/oh-my-zsh
 
 # Set name of the theme to load --- if set to "random", it will
 # load a random theme each time Oh My Zsh is loaded, in which case,
